### ISSUE TITLE
Failed to load resource: the server responded with a status of 403 ()

### ISSUE THREAD

**Issue by hopk1ns87** [stevenschobert/instafeed.js#550]
Getting this error on 2 separate websites:
www.goalminenyc.com 
www.zoganic.com

Any idea why these arent displaying images, been driving me crazy

**Comment by saturday1** [comment]
Same problem for me.

**Comment by LucaThemes** [comment]
I can confirm that. Recreating a client doesn't work.

**Comment by saturday1** [comment]
I read this thread: https://github.com/stevenschobert/instafeed.js/issues/549

It has something with scaling to do. Something has changed. But changing image size to 150px seems to work, but the images is really low res... So I hope for a better solution by someone smart soon.

I fixed the problem temporary by changing this code:

`
url: e.images.thumbnail.url.replace('150x150', '640x640'),
width: 640,
height: 640`

into this:

`
url: e.images.thumbnail.url.replace('150x150', '150x150'),
width: 150,
height: 150`

**Comment by LucaThemes** [comment]
Actually broken links are missing this part:
![insta](https://user-images.githubusercontent.com/14250332/34869620-60cc5e96-f77f-11e7-819f-56b9d0ee7755.jpg)

**Comment by saturday1** [comment]
@LucaThemes ok, but even with the missing part it does not work (for me at least)

I made the link like this: `/s640x640/**sh0.08**/e35/`

**Comment by LucaThemes** [comment]
Neither is for me, I've removed the part as it's irrelevant now :) 

You are onto something, for now it looks like accounts with non-square images are affected. I'm testing few more things to be able to draw more conclusions.

**Comment by saturday1** [comment]
@LucaThemes So then I removed my comment. :)

Great, let me know if you find something out. I have tried to google for what changes are done by Instagram, but I have not found anything.

**Comment by LucaThemes** [comment]
I have almost finished writing a fix, but then realized:

from now on it seems to be impossible to get square image out of original portrait. API call always results in either 'p' or 's' in front of image name, which stands for "square" and "portrait". Changing one with another will stop the image work (even though there are the same size/height provided in image address, API call seems to pull native size anyway, even if it's portrait).

In other words:
**s150x150** changed to **s320x320** will not work if the image is portrait. It has to be **p320x320** which is now pointless as image will be potrait indeed.

Solution:
I'll try to write my own template that still shows images as square in a moment.

**Comment by saturday1** [comment]
@LucaThemes Aha! Great works!

**Comment by LucaThemes** [comment]
@saturday1 

Oh well, I haven't had chance to test landscape images :) It looks like it's still "s", otherwise it should be yet another obstacle for us. And yes, it looks like even though image is not square, it's address will always display even width/height, like in portrait images.. At least today, until there's yet another helpful update.

**Comment by saturday1** [comment]
@LucaThemes Ah, now I see all available sizes. :) But still no clue how to handle the problem though. Did you find any solution?

**Comment by LucaThemes** [comment]
@saturday1 Kinda, I've provided template code that uses Insta pictures as background of square container. There's no longer need to resize pictures.

**Comment by saturday1** [comment]
Ok. Yeah, I see, that is a solution ofc. Overflow: hidden etc. Thanks. Have a great weekend!

**Comment by LucaThemes** [comment]
@saturday1 There's even no need for overflow, background will fit inside container, make sure to use "background-size: cover" property to ".insta-thumb a {" class :)

Have a great one as well!

**Comment by saturday1** [comment]
@LucaThemes Ah, ofc, you are right. I'll have to blame it is late friyay afternoon. ;)

**Comment by dec23rd1986** [comment]
Hi all, I can't seem to figure out how to get the client ID for my client or token. Per all the examples I have seen they are using their own Instagram ID. Can someone tell me how to obtain this? Do I have to have the client request it and then send it to me?

I'm trying to put an Instagram feed carousel on the homepage of their website.

**Comment by LucaThemes** [comment]
Hi @dec23rd1986, client ID is the first number you're gonna see after creating a client on https://www.instagram.com/developer/. 

You may need to open new thread, this one is related to completely different problem :)

**Comment by saturday1** [comment]
Just for sharing: I found out that `background-position: center` and `background-size: 100%` fitted me the best using this template:

```
<div class="insta-obj">
  <div class="img-container">
    <a href="{{link}}" class="img-link" data-type="{{model.type}}" style="background: url(\'{{image}}\') no-repeat;"></a>
  </div>
  <div class="meta-data">
    <span class="likes">{{likes}}</span>
    <span class="comments">{{comments}}</span>
    <p>{{caption}}</p>
  </div>
</div>
```

Result is visible here: https://arenagodemo.axiell.com/web/arena

**Comment by surjithctly** [comment]
I'm fetching **low resolution** instead of **thumbnail** and it works fine for me. 

https://demo.web3canvas.com/themeforest/unisco/

**Comment by LucaThemes** [comment]
@surjithctly Welll, true, but you are already using a background technique while most of us were simply displaying raw images :)

**Comment by surjithctly** [comment]
@LucaThemes  Yes. Several techniques posted on the #527 issue.

And my solution works fine for me: https://github.com/stevenschobert/instafeed.js/issues/527#issuecomment-326204764

**Comment by LucaThemes** [comment]
Just a hint - if you need to make your Insta images responsive, try to use my CSS solution :)
https://github.com/stevenschobert/instafeed.js/issues/550#issuecomment-357213232

**Comment by JamoDevNich** [comment]
The recent url signing enforced by Instagram might be messing with custom resolutions; the fixes discussed in this thread may help: https://github.com/stevenschobert/instafeed.js/issues/549
